src/foam/u2/search/SearchBar.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.search',
+  name: 'SearchBar',
```

Comment:
How is this different from TextSearchView.js?

Reply:
We also have SearchField if you dont need the predicate coming from TextSearchView

Reply:
Did not know searchfield existed, i'll just use that then, thanks